src/append/rolling_file/policy/compound/trigger/.#time.rs

```diff
@@ -0,0 +1 @@
+sebo@sebo-pc.14422:1583722677
```

Comment:
What is this?

Reply:
Sorry, I'll delete it. It's a temple file created by emacs.